rfcs/20220926-evolution.md

```diff
@@ -0,0 +1,102 @@
+# StableHLO Evolution Process
+
+## Motivation
+
+HLO/MHLO are supported by a wide variety of ML frameworks and compilers,
+including IREE, JAX, ONNX, PyTorch, TensorFlow and XLA. With StableHLO, we are
+aiming to build on this success and create an amazing portability layer between
+ML frameworks and ML compilers.
+
+To that end, we are establishing the StableHLO evolution process to provide
+StableHLO users with well-defined means of following and influencing the
+evolution of the StableHLO opset and accompanying infrastructure (the CHLO
+opset, bytecode serialization/deserialization, etc).
+
+## Scope
+
+This process governs the evolution of StableHLO and CHLO opsets and bytecode,
```

Comment:
I would qualify it as "minimally applying to". I would also note that general consensus in a design discussion or code review can conclude that any topic requires an RFC, at which point, this process applies.

While rarer, general consensus can also conclude that something which may technically appear to require an RFC is not of consequence and should just be dealt with by normal code review processes.

Reply:
Ok I think that something like this could be good enough.